docs/en/guide/start/quick-start.mdx

```diff
@@ -33,19 +33,19 @@ Lynx Explorer is a sandbox for trying out Lynx quickly.
 
 <PlatformTabs hashKey="explorer-platform">
 
-<PlatformTabs.Tab platform="ios-simulator">
+<PlatformTabs.Tab platform="ios">
 
-:::info
-We currently only provide pre-built binaries for the iOS simulator. If you need to run Lynx Explorer on a real iOS device, you'll need to build it from source. Please refer to the [Build Lynx Explorer for iOS](https://github.com/lynx-family/lynx/tree/develop/explorer/darwin/ios) guide.
-:::
+To run Lynx Explorer on real iOS device, download the [Lynx Go from App Store](https://apps.apple.com/us/app/lynx-go-dev-explorer/id6743227790).
+
+To run Lynx Explorer on <ins>Simulator</ins>, follow the steps below:
```

Comment:
I'll need to discuss with the team and PR folks, but personally, i'd be happy with something along these lines: 

```
We currently provide pre-built binaries for the iOS **Simulator** only.  
If you'd like to run Lynx Explorer on a phyical iOS device, you'll need to build it from source by following our [Build Guide for iOS](https://github.com/lynx-family/lynx/tree/develop/explorer/darwin/ios).

:::info
A version of Lynx Explorer is also available on the [App Store](https://apps.apple.com/us/app/lynx-go-dev-explorer/id6743227790), published by community contributors.  
While this version is not reviewed and maintained by the Lynx team, we’re thankful to the community for making it more convenient for developers to try out Lynx on iOS.
:::
```

Reply:
Sounds good. I have updated the PR to incorporate the described changes